test/environments/railsedge/config/application.rb

```diff
@@ -13,7 +13,7 @@
 module RpmTestApp
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults(8.0) # TODO: keep this number up to date (and keep this TODO)
+    config.load_defaults(8.1) # TODO: keep this number up to date (and keep this TODO)
```

Comment:
There may not be Rails 8.1 defaults yet... 
https://github.com/newrelic/newrelic-ruby-agent/actions/runs/11806613018/job/32891658488#step:12:6141 
in active_record_pg suite running on Rails 8.0